demoapp-ios-swift/DetailViewController.swift

```diff
@@ -10,34 +10,38 @@ import UIKit
 
 class DetailViewController: UIViewController {
 
+    @IBOutlet weak var beaconBlurb: UITextView!
```

Comment:
We should not define variables with  "beacon" as prefix in it - anywhere in the code. Can you change it to "serviceBlurb"?